tests/pass-dep/libc/threadname.rs

```diff
@@ -19,6 +19,10 @@ fn main() {
                 0
             } else if #[cfg(target_os = "macos")] {
                 unsafe { libc::pthread_setname_np(name.as_ptr().cast()) }
+            } else if #[cfg(target_os = "android")] {
+                // FIXME: Use PR_SET_NAME constant when https://github.com/rust-lang/libc/pull/3941 lands.
+                const PR_SET_NAME: i32 = 15;
+                unsafe { libc::prctl(PR_SET_NAME, name.as_ptr().cast::<*const c_char>()) }
```

Comment:
```suggestion
                unsafe { libc::prctl(PR_SET_NAME, name.as_ptr().cast::<libc::c_char>()) }
```
You made this cast to a pointer-to-pointer.